database/migrations/2024_09_16_093650_inactive_groups.php

```diff
@@ -0,0 +1,45 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+return new class extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        $groups = \App\Group::join('grouptags_groups', 'groups.idgroups', '=', 'grouptags_groups.group')
+            ->join('group_tags', 'grouptags_groups.group_tag', '=', 'group_tags.id')
+            ->where('group_tags.id', \App\GroupTags::INACTIVE)
+            ->get();
+
+        foreach ($groups as $group) {
+            $group->archived_at = $group->updated_at;
+
+            // Remove [INACTIVE] from the group name - this is now indicated via archived_at.
+            $group->name = str_replace('[INACTIVE]', '', $group->name);
```

Comment:
This should also remove the space, i.e. '[INACTIVE] '.

Reply:
Agree.